### PR TITLE
Update to use new PyPortal AIO image converter

### PR DESCRIPTION
**Pull request by martymcguire**
depends on https://github.com/adafruit/Adafruit_CircuitPython_PyPortal/pull/6

### CODE REVIEW

File: PyPortal_Thingiverse/thingiverse.py [hunk 1]
```diff
@@ -36,8 +37,8 @@
         print("Response is", response)
         pyportal.set_background(None)
         image_url = response[2].replace('_thumb_medium.', '_display_large.')
-        pyportal.wget(adafruit_pyportal.IMAGE_CONVERTER_SERVICE+image_url, "/cache.bmp")
-        pyportal.set_background("/cache.bmp")
+        pyportal.wget(pyportal.image_converter_url(image_url, 320, 240), "/sd/cache.bmp")
```
Comment (abachman) [inline PyPortal_Thingiverse/thingiverse.py]: not sure if it's consistent across URLs, but I had trouble with wget never returning if I didn't manually set the chunk_size low: 

```python
        pyportal.wget(pyportal.image_converter_url(DATA_SOURCE, 320, 240), "/sd/cache.bmp", chunk_size=512)
```

Comment (ladyada) [reply PyPortal_Thingiverse/thingiverse.py]: yes SD cards require a smaller chunk (due to a bug) - @martymcguire want to fix?

Comment (martymcguire) [reply PyPortal_Thingiverse/thingiverse.py]: added `chunk_size=512`. seems to work okay!

**Comment by ladyada** [comment]
plz lint :)

**Comment by martymcguire** [comment]
linted!

**Pull request merged**